pkg/api/v1beta3/conversion.go

```diff
@@ -3227,3 +2746,474 @@ func init() {
 		panic(err)
 	}
 }
+
```

Comment:
A lot of this diff is functions being relocated within the file without actual changes -- did the generator change and begin putting them in a different place?  I'd prefer to keep these functions (`convert_api_ReplicationController_To_v1beta3_ReplicationController` as an example) out of the diff if we can.

This is one of those things about code generation -- sometimes the generator changes and the whole file is a mess of diffs.  That said it looks like we can cut the diff back here to a great degree.